AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_PeriodicTests.py

```diff
@@ -0,0 +1,409 @@
+
+        # 11. Verify the changes are saved in child_material_document.
+        Report.result(
+            Tests.saved_changes_to_child_material_document,
+            material_editor_utils.get_property(
+                child_material_document_id, base_color_property_name) == child_material_document_color)
+
+        # 12. Revert original_material_document color to starting_material_document_color and save it.
+        material_editor_utils.set_property(
+            original_material_document_id, base_color_property_name, starting_material_document_color)
+        Report.result(
+            Tests.reverted_original_material_document_color,
+            material_editor_utils.get_property(
+                original_material_document_id, base_color_property_name) == starting_material_document_color)
+        original_document_saved = atom_tools_utils.save_document(original_material_document_id)
+        Report.result(
+            Tests.resaved_original_material_document,
+            original_document_saved is True)
+
+        # 13. Close all currently opened documents.
+        closed_all_material_documents = atom_tools_utils.close_all_documents()
+        Report.result(
+            Tests.reclosed_original_new_child_material_documents,
+            closed_all_material_documents is True)
+        for material_document_id in material_document_ids:
+            Report.result(
+                Tests.reclosed_original_new_child_material_documents,
+                atom_tools_utils.is_document_open(material_document_id) is False)
+
+        # 14. Open a material document referred to as "first_material_document".
+        first_material_document_id = atom_tools_utils.open_document(first_material_document)
+        Report.result(
+            Tests.opened_first_material_document,
+            atom_tools_utils.is_document_open(first_material_document_id) is True)
+
+        # 15. Change the baseColor.color property value of first_material_document to 0.0156, 0.0196, 0.0862, 1.0.
+        starting_first_material_document_color = material_editor_utils.get_property(
+            first_material_document_id, base_color_property_name)
+        first_material_document_color = azlmbr.math.Color(0.0156, 0.0196, 0.0862, 1.0)
+        material_editor_utils.set_property(
+            first_material_document_id, base_color_property_name, first_material_document_color)
+        Report.result(
+            Tests.changed_first_material_document_color,
+            material_editor_utils.get_property(
+                first_material_document_id, base_color_property_name) == first_material_document_color)
+
+        # 16. Save the first_material_document as a new material document file.
+        first_material_document_saved = atom_tools_utils.save_document(first_material_document_id)
+        Report.result(
+            Tests.saved_first_material_document,
+            first_material_document_saved is True)
```

Comment:
we should have one of these not save changes to confirm that without saving the changes are not persisted to file.

Reply:
this test was actually in error and your feedback helped me identify that this should be `save_document_as_copy` instead of `save_document`

that removes the risk of changing the original file